yeat/Snakefile

```diff
@@ -105,6 +107,7 @@ rule downsample:
             down = int((genome_size * params.coverage) / (2 * avg_read_length))
         else:
             down = params.downsample
+        random.seed(params.seed) # for unit tests
         seed = randint(1, 2**16-1)
```

Comment:
The `random.seed()` setting is the only solution that I could think of to enforce `randint` to be the same number every time this rule is ran for pytest.